packages/client/src/v2-events/features/events/useCertificateTemplateSelectorFieldConfig.test.ts

```diff
@@ -0,0 +1,211 @@
+
+function birthCertificateTemplateWithMinimumAge(
+  minimumAge: number
+): CertificateTemplateConfig {
+  return {
+    ...birthCertificateTemplateWithNoConditional,
+    conditionals: [
+      {
+        type: 'SHOW',
+        conditional: {
+          type: 'object',
+          properties: {
+            $form: {
+              type: 'object',
+              properties: {
+                'child.dob': {
+                  type: 'string',
+                  format: 'date',
+                  formatMaximum: new Date(
+                    dateToday.getFullYear() - minimumAge,
+                    dateToday.getMonth(),
+                    dateToday.getDate()
+                  ).toISOString()
+                }
+              },
+              required: ['child.dob']
+            }
+          },
+          required: ['$form']
+        } as unknown as JSONSchema
+      }
+    ]
+  }
+}
+const declaration = {
+  'mother.firstname': 'Mom',
+  'mother.surname': 'Test',
+  'mother.dobUnknown': true,
+  'mother.age': '80',
+  'mother.nationality': 'FAR',
+  'mother.idType': 'NATIONAL_ID',
+  'mother.nid': '9898989898',
+  'mother.address': {
+    country: 'ZAF',
+    addressType: 'INTERNATIONAL',
+    state: 'Western Cape',
+    district2: 'City Centre',
+    cityOrTown: 'Cape Town'
+  },
+  'mother.maritalStatus': 'SINGLE',
+  'mother.educationalAttainment': 'NO_SCHOOLING',
+  'mother.occupation': '',
+  'father.detailsNotAvailable': true,
+  'father.reason': 'Ran',
+  'informant.relation': 'MOTHER',
+  'informant.phoneNo': '0791234567',
+  'informant.email': 'mom@example.com',
+  'child.firstname': 'Kido',
+  'child.surname': 'Test',
+  'child.gender': 'male',
+  'child.dob': dateToday.toISOString(),
+  'child.reason': 'Procastination',
+  'child.placeOfBirth': 'HEALTH_FACILITY',
+  'child.birthLocation': '278fe332-e9d9-42e9-9ce3-80df8d37e97f',
+  'child.attendantAtBirth': 'PHYSICIAN',
+  'child.birthType': 'SINGLE',
+  'child.weightAtBirth': 2.5
+} as EventState
+
+let mockTemplates: CertificateTemplateConfig[] = []
+describe('useCertificateTemplateSelectorFieldConfig', () => {
+  vi.mock('@client/v2-events/hooks/useAppConfig', () => ({
+    useAppConfig: () => ({
+      certificateTemplates: mockTemplates
+    })
+  }))
+
+  beforeEach(() => {
+    vi.resetModules()
+    vi.clearAllMocks()
+    mockTemplates = []
+  })
+
+  it('should return a certificate as an option if it does not have a conditional and matches event type', async () => {
+    mockTemplates = [
+      { ...birthCertificateTemplateWithNoConditional },
+      { ...deathCertificateTemplateWithNoConditional }
+    ]
+
+    const { useCertificateTemplateSelectorFieldConfig } = await import(
+      './useCertificateTemplateSelectorFieldConfig'
+    )
+    const result = useCertificateTemplateSelectorFieldConfig(
+      'v2.birth',
+      declaration
+    ) as SelectField
+  })
+
+  it('does not include birth-certified-certificate if age is less than a year old', async () => {
```

Comment:
This logic is not static, countries can configure conditionals for birth certificates. Name your test appropriately and use describes to layout the preconditions: Something like:
```
describe('when a certificate has a conditional which should only show one year after birth', () => {
  it('should not return the certificate if within one year')
}
```